sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -1,13 +1,13 @@
 # Azure Form Recognizer client library for Python
 
-Azure Cognitive Services Form Recognizer is a cloud service that uses machine learning to extract text and table data
-from form documents. Form Recognizer is made up of the following services:
+Azure Cognitive Services Form Recognizer is a cloud service that uses machine learning to recognize text and table data
+from form documents. It includes the following main functionalities:
 
-* Custom models - Extract name/value pairs and table data from forms. These models are trained with your own data, so they're tailored to your forms.
-* Prebuilt receipt model - Extract data from USA sales receipts using a prebuilt model.
-* Layout API - Extract text and table structures, along with their bounding box coordinates, from documents.
+* Custom models - Recognize name/value pairs and table data from forms. These models are trained with your own data, so they're tailored to your forms. You can then take these custom models and recognize forms. You can also manage the custom models you've created and see how close you are to the limit of custom models your account can hold.
```

Comment:
name/value pairs is a little awkward because in the API we're using "name" as the unique identifier of a form field, and "label" to indicate the label of a form field.  Could we say "recognize field values" instead?